templates/kubelet.service

```diff
@@ -32,6 +32,8 @@ ExecStart={{coreos_kubelet_install_dir}}/kubelet-wrapper \
 
 Restart=always
 RestartSec=10
+StartLimitInterval=600
```

Comment:
I believe this should be `StartLimitIntervalSec`. Looks like this setting was renamed as such in recent systemd versions.

Reply:
Updated as per request. Also, moved both config options into the [Unit] section from the [Service] section because as per systemd documentation that is where they are supposed to be now.